interfaces/Portalicious/src/app/pages/project-payments/project-payments.page.html

```diff
@@ -3,5 +3,20 @@
   i18n-pageTitle="@@page-title-project-payments"
   [projectId]="projectId()"
 >
-  <p i18n>project-payments works!</p>
+  <ul
+    class="grid grid-cols-1 gap-4 lg:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4"
```

Comment:
Maybe as part of your refactor task, we can also refactor this into a `app-card-grid` component.